pandas/tests/frame/test_sorting.py

```diff
@@ -321,7 +326,27 @@ def test_sort_nat_values_in_int_column(self):
         assert_frame_equal(df_sorted, df_reversed)
 
         df_sorted = df.sort_values(["datetime", "float"], na_position="last")
-        assert_frame_equal(df_sorted, df_reversed)
+        assert_frame_equal(df_sorted, df)
+
```

Comment:
ok for a new issue, generally like a separate test. can you fix here. otherwise lgtm.